src/platform/datapath_raw_socket.c

```diff
@@ -371,7 +390,6 @@ CxPlatResolveRoute(
     //
     // Map the next hop IP address to a link-layer address.
     //
-    MIB_IPNET_ROW2 IpnetRow = {0};
```

Comment:
This IpnetRow variable was moved to the top of the function, but the general QUIC code style is to declare variables as late as possible.

Reply:
something leftover during undoing the previous change.